src/ts/components/playbacktogglebutton.ts

```diff
@@ -39,8 +39,10 @@ export class PlaybackToggleButton extends ToggleButton<ToggleButtonConfig> {
 
       if (player.isPlaying() || this.isPlayInitiated) {
         this.on();
+        this.getDomElement().attr('aria-label', i18n.performLocalization(i18n.getLocalizer('pause')));
       } else {
         this.off();
+        this.getDomElement().attr('aria-label', i18n.performLocalization(i18n.getLocalizer('play')));
```

Comment:
Suggestion: You could maybe split these two virtually identical lines to a private method?